src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/AddAddressModal/AddAddressModal.tsx

```diff
@@ -207,6 +207,9 @@ export const AddAddressModal: React.FC<EditContactAddressModalProps> = ({
                           {t('Location')}
                         </InputLabel>
                         <AddressLocationSelect
+                          name="location"
+                          label={t('Location')}
```

Comment:
I would be fine with the default `label` going back in the common component. But it's not a big deal either way.